www_src/blocks/image.jsx

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+var React = require("react");
+
+var Image = React.createClass({
+  statics: {
+    defaults: {
+      src: '',
+      alt: '',
+      opacity: 1,
+      borderWidth: 0,
+      borderColor: 'transparent'
+    }
+  },
+  getDefaultProps: function () {
+    return this.defaults;
+  },
+  render: function() {
+    var style = {
+      opacity: this.props.opacity,
+      borderStyle: 'solid',
+      borderWidth: this.props.borderWidth,
+      borderColor: this.props.borderColor
+    };
+    return <img style={style} src={this.props.src} alt={this.props.alt} />
```

Comment:
do we want to use the slightly more complex LoaderImage that @thisandagain created?